UnityProject/Assets/Scripts/Items/Bureaucracy/HandLabeler.cs

```diff
@@ -89,7 +97,7 @@ public void ServerPerformInteraction(InventoryApply interaction)
 		{
 			labelAmount = LABEL_CAPACITY;
 			_ = Despawn.ServerSingle(interaction.UsedObject);
-			Chat.AddExamineMsg(interaction.Performer, $"You insert the {interaction.UsedObject.Item().ArticleName.ToLower()} into the {gameObject.Item().InitialName.ToLower()}.");
+			Chat.AddExamineMsg(interaction.Performer, $"You insert the {interaction.UsedObject.AttributesOrNull().ArticleName.ToLower()} into the {gameObject.AttributesOrNull().InitialName.ToLower()}.");
```

Comment:
what is the string going to be when ``AttributesOrNull``returns null?

Reply:
that's not what I am reading in the code. It will throw an NRE if ``AttributesOrNull`` returned null.

Reply:
yeah i noticed, so i fixed it quickly

Reply:
yeah i noticed, so i fixed it quickly